src/summarizedexperiment/utils.py

```diff
@@ -0,0 +1,80 @@
+from typing import Optional, Sequence, MutableMapping
+from functools import reduce
+import numpy as np
+import pandas as pd
+import scipy.sparse as sp
+
+from .SummarizedExperiment import SummarizedExperiment
+
+
+def concat(
+    ses: Sequence[SummarizedExperiment],
+    how: str = "outer",
+    metadata: Optional[MutableMapping] = None,
+) -> SummarizedExperiment:
+    """Concatenate SummarizedExperiment objects.
+    Assume we are working with RNA-Seq experiments for now.
+
+    Parameters
+    ----------
+    ses: Sequence[SummarizedExperiment]
+        A sequence of SummarizedExperiment objects.
+    how: str
+        Method to join SummarizedExperiment objects.
+    metadata: str
+        Metadata of the new SummarizedExperiment.
+
+    Returns
+    -------
+    concat_se: SummarizedExperiment
+        The concatenated SummarizedExperiment.
+    """
```

Comment:
I don't mind the numpy style docstrings, but just to be consistent with the rest of the package, lets stick to google style.